python/langsmith/client.py

```diff
@@ -5350,13 +5350,20 @@ def push_prompt(
         """
         # Create or update prompt metadata
         if self._prompt_exists(prompt_identifier):
-            self.update_prompt(
-                prompt_identifier,
-                description=description,
-                readme=readme,
-                tags=tags,
-                is_public=is_public,
-            )
+            if not (
```

Comment:
nit: easier to read if we flip the conditions:

```
if (all this stuff is none) then create prompt, else update

Reply:
we don't want to create the prompt at all in this case, but if we have an existing prompt and no metadata is passed we don't want to call update_prompt since there's nothing to update

this is just the case where we're updating the manifest and not the metadata

Reply:
I'm just saying it's hard to read a double negative (not x is None); the logic is right